tasks/bbb_secret.yml

```
@@ -0,0 +1,15 @@
+---
+- name: set bbb_secret as fact
+  block:
+    - name: register bbb secret
+      become: true
+      command: bbb-conf --secret
+      changed_when: false
+      register: result
+
+    - name: parse bbb secret
+      set_fact:
+        bbb_secret: "{{ result.stdout | regex_search('Secret: ([a-zA-Z0-9]*)', multiline=True) | regex_replace('Secret: ') }}"
+        cacheable: true
+      when: result.rc == 0
+  when: bbb_secret is not defined
```

```suggestion
  when: bbb_secret is not defined

```
Add new line in end of file.